examples/paintbrush.md

```diff
@@ -0,0 +1,306 @@
+# 🖌️ Paintbrush
+
+This example shows how to create a paintbrush in Minecraft using the Bookshelf library. The paintbrush changes a block's material to match the item in the player's hand, while preserving important properties like orientation, waterlogging, and other block states. For instance, if the player holds oak planks, birch stairs will transform into oak stairs.
+
+---
+
+## 🎯 What We'll Build
+
+Our paintbrush system will:
+1. Detect when a player interacts with a block using the brush
+2. Determine the block type from the item in the player's hand
+3. Identify the targeted block
+4. Replace the targeted block's material with that of the held item
+
+---
+
+## 📦 Requirements
+
+Before we start, make sure you have:
+- Minecraft Java Edition 1.21.4 or later
+- Bookshelf `bs.block` and `bs.view` modules, see [Quickstart](https://docs.mcbookshelf.dev/en/latest/quickstart.html) for installation
+  - `@require bookshelf.module.block`
+  - `@require bookshelf.module.view`
+- Basic knowledge of datapack development, including functions, tags, and scoreboards
+
```

Comment:
Perhaps we should add a download to an empty datapack template to quick start, or a "- a ready-to-use empty datapack"?
Because we directly start by "create a function".

Reply:
Ideally a template would be best, could be what we talked about a few days ago (a template with beet and bookshelf preinstalled). But I think it's out of this scope and something for later, also it may be too complex for small experimentation, so i'll just use a new bullet item for now